templates/jobs/main.html

```diff
@@ -23,12 +23,10 @@
     <section id="community">
         <div class="row">
             <div class="col-md-9">
-                <h2>Community</h2>
+                <h2>Django Girls Community</h2>
 
-                <p>Here, you will find information about job offers for
-                    entry-level positions, meetups, local groups and other
-                    things that will help you get going with becoming a
-                    fully-fledged Django programmer.</p>
+                <p>If you find yourself on this page, you probably recently completed Django Girls tutorial — either on one of our workshops, at the event in your city or alone at home.</p>
```

Comment:
If you find yourself on this page, you probably recently completed the Django Girls tutorial — either at one of our workshops, at an event in your city or at home on your own.